charts/catalog/templates/apiregistration.yaml

```diff
@@ -13,6 +13,11 @@ spec:
   service:
     namespace: {{ .Release.Namespace }}
     name: {{ template "fullname" . }}-apiserver
-  caBundle: {{ .Values.apiserver.tls.ca }}
+  caBundle: {{ (.Files.Get .Values.apiserver.tls.caFileName) | b64enc }}
+  {{ if .Capabilities.APIVersions.Has "apiregistration.k8s.io/v1alpha1" -}}
   priority: 100
+  {{ else if .Capabilities.APIVersions.Has "apiregistration.k8s.io/v1beta1" -}}
+  groupPriorityMinimum: 10000
+  versionPriority: 20
+  {{- end }}
```

Comment:
@MHBauer I will not be able to split this out today. Please feel free to pull this into another PR, and I'll rebase as necessary.